CrazyCanvas/Source/States/BenchmarkState.cpp

```diff
@@ -305,12 +314,13 @@ bool BenchmarkState::OnPacketReceived(const LambdaEngine::NetworkSegmentReceived
 				.pCameraDesc 		= &cameraDesc
 			};
 
-			for (uint32 playerNr = 0; playerNr < 9; playerNr++)
+			for (uint32 playerNr = 0; playerNr < playerCount; playerNr++)
 			{
 				// Create a 3x3 grid of players in the XZ plane
 				createPlayerDesc.Position.x			= -3.0f + 3.0f * (playerNr % 3);
 				createPlayerDesc.Position.z			= -3.0f + 3.0f * (playerNr / 3);
-				createPlayerDesc.PlayerNetworkUID	+= (int32)playerNr;
+				createPlayerDesc.PlayerNetworkUID	+= 2;
+				createPlayerDesc.WeaponNetworkUID	+= 2;
```

Comment:
Why do we need to increment by 2?

Reply:
`PlayerNetworkUID` starts at 0, `WeaponNetworkUID` at 1. If both are incremented by 2, players and weapons will never be the same.